sklearn/metrics/_classification.py

```diff
@@ -2948,6 +2948,14 @@ def hinge_loss(y_true, pred_decision, *, labels=None, sample_weight=None):
     return np.average(losses, weights=sample_weight)
 
 
+@validate_params(
+    {
+        "y_true": ["array-like"],
+        "y_prob": ["array-like"],
```

Comment:
Could you also update the documentation below to change `array` to `array-like`.

Reply:
Absolutely, I just now added the updates.